app/Http/Controllers/TCAcontroller.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\taskcrudapp;
+use Illuminate\Http\Request;
+
+class TCAcontroller extends Controller
```

Comment:
Controller name should be `TaskController`. Never abbreviate! coz, if you come and look this code after 2 months, you will be scratching your head thinking what does that mean! :smile:

Reply:
understood